test/commands/pull.js

```diff
@@ -0,0 +1,127 @@
+                        + '"first_number": {"label": "First number","format": "number"},'
+                        + '"second_number": {"label": "Second number","format": "number"}}}}}}}}',
+                },
+                {
+                    name: 'src',
+                    folder: true,
+                },
+                {
+                    name: 'src/__init__.py',
+                    format: 'TEXT',
+                    content: '',
+                },
+                {
+                    name: 'README.md',
+                    format: 'TEXT',
+                    content: '# Start with Python template\n\nA simple Python example of core Actor and Apify SDK features. It reads and validates user input with schema, computes a result and saves it to storage.\n\n## Getting Started\n\n### Install Apify CLI\n\n#### Using Homebrew\n\n```Bash\nbrew install apify/tap/apify-cli\n```\n\n#### Using NPM\n\n```Bash\nnpm -g install apify-cli\n```\n\n### Create a new Actor using this template\n\n```Bash\napify create my-python-actor -t python-start\n```\n\n### Run the Actor locally\n\n```Bash\ncd my-python-actor\napify run\n```\n\n## Deploy on Apify\n\n### Log in to Apify\n\nYou will need to provide your [Apify API Token](https://console.apify.com/account/integrations) to complete this action.\n\n```Bash\napify login\n```\n\n### Deploy your Actor\n\nThis command will deploy and build the Actor on the Apify Platform. You can find your newly created Actor under [Actors -> My Actors](https://console.apify.com/actors?tab=my).\n\n```Bash\napify push\n```\n\n## Documentation reference\n\nTo learn more about Apify and Actors, take a look at the following resources:\n\n- [Apify SDK for Python documentation](https://docs.apify.com/sdk/python)\n- [Apify Platform documentation](https://docs.apify.com/platform)\n- [Join our developer community on Discord](https://discord.com/invite/jyEM2PRvMU)\n',
+                },
+                {
+                    name: 'requirements.txt',
+                    format: 'TEXT',
+                    content: '# Add your dependencies here.\n# See https://pip.pypa.io/en/latest/reference/requirements-file-format/\n# for how to format them\napify ~= 1.0.0\n',
+                },
+            ],
+        },
+    ],
+};
+
+const TEST_ACTOR_GITHUB_GIST = {
+    name: ACTOR_NAME,
+    isPublic: false,
+    versions: [
+        {
+            versionNumber: '0.0',
+            sourceType: 'GITHUB_GIST',
+            buildTag: 'latest',
+            gitHubGistUrl: 'https://gist.github.com/DennisKallerhoff/32f1efd686e11f6a05f1af87bddb1f1a',
+        },
+    ],
+};
+
+const TEST_ACTOR_GIT_REPO = {
+    name: ACTOR_NAME,
+    isPublic: false,
+    versions: [
+        {
+            versionNumber: '0.0',
+            sourceType: 'GIT_REPO',
+            buildTag: 'latest',
+            gitRepoUrl: 'https://github.com/HonzaTuron/baidu-scraper#multipage',
+        },
+    ],
+};
+
+describe('apify pull', () => {
```

Comment:
It would be good to test also the `apify pull` without argument when already in an actor folder.